json-rpc/src/util.rs

```diff
@@ -54,6 +54,8 @@ macro_rules! register_rpc_method {
     };
 }
 
+const SDK_LANGS: &[&str] = &["java", "python", "typescript", "rust", "cluster-test"];
```

Comment:
Why call out cluster-test specifically? AFAIK cluster test actually should be using the "rust" sdk. Also what about go?